add support for muliple public ssh key (#96)
scalar-labs/scalar-terraform: examples/aws/README.md

# If needed start ssh-agent using: eval $(ssh-agent -s)
$ ssh-add example_key

> # to add access to your teammate, you can add the public ssh key under additional_public_keys file (one line, one key)

Reviewer: ```suggestion
# Optionally, you may want to create a file named `additional_public_keys` that contains multiple ssh public keys (one key per line) to allow other admins to access nodes created by the following `terraform apply`.
```

> # the file should be like

Reviewer: ```suggestion
# the file should look like below.
```

# cat examples/aws/network/additional_public_keys

> # ssh-rsa AAAAB3Nza..... teammate1

Reviewer: ```suggestion
# ssh-rsa AAAAB3Nza..... admin1
```

> # ssh-rsa...... teammate2

Reviewer: ```suggestion
# ssh-rsa...... admin2
```

> 

Reviewer 2: Let's delete one blank line.
